lib/lotus/cli.rb

```diff
@@ -79,6 +79,19 @@ def new(name = nil)
       end
     end
 
+    desc 'generate', 'generates a new action'
+    method_option :path,                desc: 'applications path',        type: :string, default: 'apps'
```

Comment:
@joneslee85 On the other hand, if we remove that option, we're forced to hardcode that value (`"apps"`) in `Lotus::Commands::Generate` or `Lotus::Generators::Action`. Isn't it better to leave some flexibility to devs?

Please let me know. :smiley_cat: